docs/reference/cli/resource/set.md

```diff
@@ -31,17 +31,27 @@ This subcommand can only be invoked for command-based DSC Resources that define
 of their resource manifest. If this subcommand is called for a resource that doesn't define a set
 operation, DSC raises an error.
 
-The subcommand's behavior depends on the value of the `set.preTest` option in the resource
-manifest:
-
-- If the resource's manifest doesn't define the `set.preTest` key as `true`, DSC invokes the
-  resource's test operation to determine whether a set operation is required.
-
-  If the instance is already in the desired state, DSC doesn't invoke the set operation. If the
-  instance isn't in the desired state, DSC invokes the resource's set operation with the desired
-  state as input.
-- If the resource's manifest defines the `set.preTest` key as `true`, DSC invokes the resource's
-  set operation without testing the resource state first.
+> [!IMPORTANT]
+> The `dsc resource set` command always invokes the `set` operation for the resource. Resources
+> may, but aren't required to, implement logic that pretests an instance for the `set` operation.
+>
+> This is different from how [dsc config set][02] works, where DSC always tests an instance, either
+> synthetically or by invoking the `test` operation for the resource, and only invokes `set` for an
+> instance if it's not in the desired state.
+>
+> If a resource doesn't implement pretest, it enforces the desired state regardless of the current
+> state when this command executes. If a resource does implement pretest, it only modifies the
```

Comment:
Perhaps it'd be better to say something like: If a resource indicates it implements pretest, then the expectation is that the resource determines if any modification of configuration is required even if a `set` operation is requested.

Reply:
Reorganized the notice a little bit to move the manifest information above the impact and reworded it around expected behavior.